microsoft-edge/progressive-web-apps-chromium/how-to/handle-urls.md

```diff
@@ -15,6 +15,9 @@ Native applications on many operating systems can handle links. Native applicati
 Progressive Web Apps (PWAs) can also handle links in a similar way.
 
 
+<!-- although the present filename is handle-urls.md, this file was repurposed to cover link handling rather than URL handlers.  see also [PWAs as URL Handlers](https://web.dev/pwa-url-handler/) -->
```

Comment:
```suggestion
<!-- although the present filename is handle-urls.md, this file was repurposed to cover link handling rather than URL handlers. The PWA URL handlers feature was removed from Chromium.  see also [PWAs as URL Handlers](https://web.dev/pwa-url-handler/) -->
```

Reply:
result:
```
<!--
link handling vs. url handling:
Although the present filename is handle-urls.md, this file was repurposed to cover link handling rather than URL handlers.
The PWA URL handlers feature was removed from Chromium.
See also [PWAs as URL Handlers](https://web.dev/pwa-url-handler/).
-->
```